kin-sdk/kin-sdk-lib/src/main/java/kin/sdk/TransactionBuilder.java

```diff
@@ -0,0 +1,90 @@
+package kin.sdk;
+
+import kin.base.KeyPair;
+import kin.base.Memo;
+import kin.base.Operation;
+import kin.base.TimeBounds;
+import kin.base.Transaction;
+import kin.base.Transaction.Builder;
+import kin.base.TransactionBuilderAccount;
+
+public class TransactionBuilder {
+
+    private final Builder builder;
+    private final KeyPair account;
+
+    /**
+     * Construct a new transaction builder.
+     *
+     * @param account The source account for this transaction. This account is the account
+     * @param sourceAccount The source account for this transaction. This account is the account who will use a sequence
+     * number. When build() is called, the account object's sequence number will be incremented.
+     */
+    TransactionBuilder(KeyPair account, TransactionBuilderAccount sourceAccount) {
+        this.account = account;
+        builder = new Builder(sourceAccount);
+    }
+
+    public int getOperationsCount() {
+        return builder.getOperationsCount();
+    }
+
+    /**
+     * Adds a new <a href="https://www.stellar.org/developers/learn/concepts/list-of-operations.html"
```

Comment:
refer to the relevant classes/instructions what can be added here instead of to stellar docs

Reply:
Its is a wrapper so i just copied the stellar docs.
I can go into the stellar docs and try to copy some summary of each method, what do you think?

Reply:
remove refer to stellar docs